src/components/labelPrinting/LabelPrintingForm.vue

```diff
@@ -181,71 +182,77 @@ const printerOptions = computed(() => {
  * Creates a computed property to get the label type
  * Created from the selected label type
  * @returns {Object} label type
+ * it would make sense to move this to the print job but it needs to be reactive. Not sure how to do that.
  */
 const labelType = computed(() => {
-  return LabelTypes[form.labelType]
+  return LabelTypes[labelOptions.labelTypeKey]
 })
 
 /**
  * Creates a computed property to get the suffix options
  * @returns {Array} suffix options
  */
-const suffixOptions = computed(() => {
-  return createSuffixDropdownOptions(SuffixList)
+const workflowDropdownOptions = computed(() => {
+  return createWorkflowDropdownOptions(WorkflowList)
 })
 
 /**
  * Creates a computed property to get the suffix items
  * @returns {Array} suffix items
  */
-const suffixItems = computed(() => {
-  return createSuffixItems(SuffixList)
+const workflowOptions = computed(() => {
+  return createWorkflowOptions(WorkflowList)
 })
 
 /**
- * Creates a computed property to get the labels
+ * Creates a computed property to get the labels (workflow barcode items)
  * @returns {Array} labels
  */
-const labels = computed(() => {
+const workflowBarcodeItems = computed(() => {
   const date = getCurrentDate()
-  const suffixItem = suffixItems.value[form.suffix]
+  // without this we get an undefined error
+  const workflowItem = workflowOptions.value[labelOptions.suffix] || NullWorkflowItem
 
   // it is possible for there to be no barcodes so we need to add a guard
-  // we filter to remove an nulls
+  // we filter to remove any nulls
   const splitSourceBarcodeList =
-    form.sourceBarcodeList?.split(/\r?\n|\r|\n/g).filter((b) => b) || []
+    printJob.sourceBarcodeList?.split(/\r?\n|\r|\n/g).filter((b) => b) || []
 
-  return createLabelsFromBarcodes({
+  const workflowListType = WorkflowListType({
     sourceBarcodeList: splitSourceBarcodeList,
     date,
-    suffixItem,
-    numberOfLabels: form.numberOfLabels,
+    workflowItem,
+    numberOfLabels: printJob.numberOfLabels,
   })
+
+  return workflowListType.createWorkflowBarcodeItemList()
 })
 
 /**
  * Creates a method to print labels
  * @returns {Object} success or failure message
  */
 const printLabels = async () => {
-  const { success, message = {} } = await printingStore.createPrintJob({
-    printerName: form.printerName,
-    labels: labels.value,
-    copies: form.copies,
-    labelTemplateName: labelType.value.labelTemplateName,
+  printJob.createLabels({
+    barcodeItems: workflowBarcodeItems.value,
+    createLabelFn: createLabelFns[labelType.value.labwareType],
   })
+  // it would be better if this was reactive but couldn't get it to work
+  printJob.labelType = labelType.value
```

Comment:
Not sure, but I think binding this to a dropdown model might make this automatically set to the value.